docs/usage/configuration/config_documentation.md

```diff
@@ -230,6 +230,21 @@ Example configuration:
 serve_server_wellknown: true
 ```
 ---
+Config option: `extra_well_known_content`
+
+This option allows server runners to add arbitrary key-value pairs to the [client-facing `.well-known` response](https://spec.matrix.org/latest/client-server-api/#well-known-uri).
+
+If this option is provided, it parses the given yaml to json and 
+serves it on `/.well-known/matrix/client` endpoint
+alongside the standard properties.
+
+Example configuration:
+```yaml
+extra_well_known_content: 
+  option1: value1
+  option2: value2
+```
+---
```

Comment:
Can we rename this to `extra_well_known_client_content` please? (To distinguish from /.well-known/server)

Also, can we add a sentence like this to the first paragraph?

```
Note that the `public_baseurl` config option must be provided  for a response to serve `/.well-known/client` at all.
```